### ISSUE TITLE
Staging testing - Borrow spotlight login prompt

### ISSUE THREAD

**Issue by sschiessl-bcp** [blockchainprojects/bitshares-ui#78]
Original issue #2438

Estimate 1

**Comment by VerevkinAlexander** [comment]
@sschiessl-bcp 
I could not reproduce the issue. 

**Steps I took:**

- Not loged in, go through the 'Collateralized Debt Position' wizard.
- Choose the SmartCoin
- Click on "borrow" button.
- Click on "update position" button.

**Result:** 
"login" modal is popes up.

Environment:
* OS: macOS 10.14.3 and Win 10
* Chrome and Mozilla latest

**Comment by sschiessl-bcp** [comment]
Have you tried local wallet as well?

Clear browser cache and try, then import a local wallet and try.

**Comment by sschiessl-bcp** [comment]
@VerevkinAlexander ping

**Comment by VerevkinAlexander** [comment]
@sschiessl-bcp 
No I have not. Thanks, I'll try as soon as possible.

**Comment by VerevkinAlexander** [comment]
@sschiessl-bcp 
I have created a local wallet containing an account. Then I took the next steps:

Log in with that local wallet.
Lock it.
Go to Spotlight > Collateralized Debt Position  https://develop.bitshares.org/#/borrow
Go through Collateralized Debt Position wizard.
Select a smartcoin.
Сlick on the "Borrow" button.
Click on "Update" button

Result: "login" popup is shown on the screen.
Setup: Win10, Chrome/Firefox/Opera last versions.
So, I have not been able to reproduce this issue.
